src/constraints/storage_constraints.jl

```diff
@@ -26,9 +26,9 @@ end
 
 function add_general_storage_dispatch_constraints(m, p, b; _n="")
     # Constraint (4a): initial state of charge
-	@constraint(m,
-        m[Symbol("dvStoredEnergy"*_n)][b, 0] == p.s.storage.attr[b].soc_init_fraction * m[Symbol("dvStorageEnergy"*_n)][b]
-    )
+	# @constraint(m,
```

Comment:
We can't just get rid of this constraint entirely because the constraint setting init and final SOC equal is only applied to elec storage (and only when require_start_and_end_charge_to_be_equal is true, though that was my change)

Reply:
fixed